packages/yarnpkg-pnp/sources/loader/makeApi.ts

```diff
@@ -100,7 +100,7 @@ export function makeApi(runtimeState: RuntimeState, opts: MakeApiOptions): PnpAp
   }
 
   function trace(entry: ReturnType<typeof makeLogEntry>) {
-    const colors = process.stderr?.hasColors() ?? process.stdout.isTTY;
+    const colors = process.stderr?.hasColors?.() ?? process.stdout.isTTY;
```

Comment:
This was undefined when I had the vscode debugger attached to the tsserver of another vscode instance